server/server.js

```diff
@@ -90,6 +90,6 @@ app.get('/server/status', (req, res) => {
   res.status(200).send(outputJson);
 });
 
-app.get('*', function(req, res, next) {
+app.get('*', function(req, res) {
   res.sendFile(path.join(__dirname, '../client/build', 'index.html'));
```

Comment:
I know you didn't do this, but can you fix this line up? You can clearly see how noob i was last year. Gotta fix broken windows as we see them!